Example.ImGui/Program.cs

```diff
@@ -20,14 +21,19 @@ static void Main(string[] args)
             ImGuiController controller = null;
             GL gl = null;
             IInputContext inputContext = null;
-            
+
+            var projectDir = Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory!, @"..\..\.."));
+            var fontPath = Path.Combine(projectDir, "segoeui.ttf");
```

Comment:
Redistributing this font is unfortunately illegal per Microsoft's license on the Segoe UI font. Consider loading it from the Windows fonts directory (if present) or use a suitable alternative (my favourite is [**Open Sans**](https://fonts.google.com/specimen/Open+Sans), it looks like Segoe UI but is licensed under Apache 2.0)

Reply:
Lol, never knew about it, will replace it with Open Sans.